docs/machine-learning/tutorials/sentiment-analysis.md

```diff
@@ -434,7 +434,6 @@ The model is saved to C:\Tutorials\SentimentAnalysis\bin\Debug\netcoreapp2.1\Dat
 Sentiment: This was a very bad steak | Prediction: Negative | Probability: 0.4641322
 =============== End of Predictions ===============
 
-
```

Comment:
I'm afraid this might be like this in the sample @JRAlexander?

Reply:
Let me know if you want it reverted. It was just a brute force Find/Replace so this actually shouldn't be picked up by the linting inside the code fence anyway

Reply:
Yes, please. Thanks, @nschonni!

Reply:
Done